cypress/e2e/SuccessContent.cy.ts

```diff
@@ -0,0 +1,131 @@
+import { StatusCodes } from 'http-status-codes';
+
+import { API_ROUTES } from '@graasp/query-client';
+
+import { SIGN_IN_PATH, SIGN_UP_PATH } from '../../src/config/paths';
+import {
+  BACK_BUTTON_ID,
+  RESEND_EMAIL_BUTTON_ID,
+  SUCCESS_CONTENT_ID,
+} from '../../src/config/selectors';
+import { MEMBERS } from '../fixtures/members';
+
+describe('Success Content Validation', () => {
+  it('Sign In - Back Button', () => {
```

Comment:
You can structure your tests with `describe` blocks. In your case I would have `describe('Sign In')` and `describe('Sign Up')`